src/commonMain/kotlin/exchange.dydx.abacus/state/manager/Enviroment.kt

```diff
@@ -455,6 +458,7 @@ class V4Environment(
             val squidIntegratorId = parser.asString(data["squidIntegratorId"])
             val chainName = parser.asString(data["chainName"])
             val chainLogo = parser.asString(data["chainLogo"])
+            val rewardsHistoryStartDateMs = parser.asString(data["rewardsHistoryStartDateMs"]) ?: return null
```

Comment:
[q] do we want to early return null here? maybe it's worth just having a default value (180 days maybe?)

Reply:
oh good point 🤔 ideally it should never be null, but probably not worth breaking/early returning on it - will make change